easy-ui-react/src/TabNav/TabNavItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ComponentProps, ElementType, ReactNode, useRef } from "react";
+import { useHover } from "react-aria";
+import { classNames } from "../utilities/css";
+import { useTabNav } from "./context";
+import { useIndicatorSizing } from "./useIndicatorSizing";
+import { useScrollIntoViewIfNeeded } from "./useScrollIntoViewIfNeeded";
+
+import styles from "./TabNavItem.module.scss";
+
+export const SCROLL_PADDING = 32;
+
+type TabNavItemProps<T extends ElementType = "a"> = ComponentProps<T> & {
+  /** Override the default element with a custom one to provide unique behavior. Useful for client-side navigation link components in app frameworks. */
+  as?: T;
+
+  /** The children of the <TabNav.Item> element. */
```

Comment:
I think most other places we surround this with backticks - noticed this and a few other in this file were displaying in the normal font in Storybook.